src/main/resources/logback-test.xml

```diff
@@ -15,10 +15,19 @@
   <logger name="LogService" level="DEBUG"></logger> <!-- Restlet access logging -->
   <logger name="net.floodlightcontroller" level="INFO"/>
   <logger name="org.sdnplatform" level="INFO"></logger>
+  <logger name="org.sdnplatform.sync.internal.config.SyncStoreCCProvider" level="TRACE"></logger>
```

Comment:
@tulioalberton In commits to master, let's reduce all logs to INFO to not clutter stdout for average users :-)